tests/test_date.py

```diff
@@ -82,6 +82,35 @@ def test_in(self):  # Same as "from now"
         self.assert_date('in 45 minutes', datetime(2017, 6, 16, 20, 22, 22))
         self.assert_date('in 45 seconds', datetime(2017, 6, 16, 19, 38, 7))
 
+    def test_since(self):
+        self.assert_date('since last may', datetime(2017, 5, 1))
+        self.assert_date('since last month', datetime(2017, 5, 1, 19, 37, 22))
```

Comment:
no other time do we include the time as well. For consistency this should either => date time (2017,5,1)

Otherwise, since last month & week as well as since this month, week, year, yesterday, etc should also include an exact time.